docs/applications/librespeed.mdx

```diff
@@ -6,6 +6,36 @@ sidebar_label: Librespeed
 
 Librespeed is a very lightweight Speedtest implemented in Javascript, using XMLHttpRequest and Web Workers. No Flash, No Java, No Websocket, No Bullshit.
 
-import MissingDocs from '../snippets/missingdocs.md';
+## Initial Setup
 
-<MissingDocs />
\ No newline at end of file
+Installing Librespeed is easy. Simply issue the following command from SSH:
+
+```bash main
+sudo box install librespeed
+```
+
+This command will configure Librespeed.
+
+## How to Access
+
+Once setup, Librespeed will be available at the link `https://<hostname.ltd>/librespeed`
+
+## Service Management
+
+Like all box configured applications, you can manage Librespeed via SSH with box with start, stop, restart, enable and disable commands.
+
+import SystemdTabs from '../snippets/systemdtabs.mdx';
+
+<SystemdTabs service="librespeed"/>
+
```

Comment:
There is no systemd service for librespeed, it's just a page served through nginx 😉

Reply:
🥸

Reply:
🤡 I'll fix this tomorrow lol